src/V3TSP.h

```diff
@@ -34,7 +34,7 @@ class TspStateBase VL_NOT_FINAL {
     // This is the cost function that the TSP sort will minimize.
     // All costs in V3TSP are int, chosen to match the type of
     // V3GraphEdge::weight() which will reflect each edge's cost.
-    virtual int cost(const TspStateBase* otherp) const = 0;
+    virtual int cost(const TspStateBase* otherp) const VL_PURE = 0;
```

Comment:
In case of `V3VariableOrder`, cost function uses static variable `ExecMTask::s_nextId` thus I marked it `VL_MT_SAFE`.
I also added missing annotations to the function definitions.